packages/block-editor/package.json

```diff
@@ -54,10 +55,11 @@
 		"@wordpress/wordcount": "file:../wordcount",
 		"classnames": "^2.2.5",
 		"css-mediaquery": "^0.1.2",
+		"css-tree": "^1.0.0-alpha.39",
```

Comment:
Why is this using an "alpha" version? can we use the latest stable instead?

Reply:
Yes, this doesn't make sense `css-tree` is in use for many years and very stable, see this issue https://github.com/csstree/csstree/issues/134.